### PR TITLE
Get Alice closer to accessing the services page.

### PR DESCRIPTION
**Pull request by rcorral**
This gets Alice a bit closer to accessing DCOS UI.
We still need to figure out how we redirect the a user who doesn't have access to the dashboard.

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @daiyiwhale

File: src/js/events/MarathonActions.js [hunk 1]
```diff
@@ -33,7 +33,7 @@ module.exports = {
 
   createGroup: function (data) {
     RequestUtil.json({
-      url: `${Config.rootUrl}/marathon/v2/groups`,
+      url: `${Config.rootUrl}/service/marathon/v2/groups`,
```
Comment (kennyt) [inline src/js/events/MarathonActions.js]: do we wanna pull out `/service/marathon/v2` into a constants file so we don't have to do this whole dance again?

**Comment by MatApple** [comment]
retest this please

File: src/js/events/MarathonActions.js (hunk 1 above)
Comment (rcorral) [inline src/js/events/MarathonActions.js]: Done.

**Comment by rcorral** [comment]
img

**Pull request merged**